src/main/webapp/projectReview/ProjectReview.js

```diff
@@ -514,39 +539,40 @@ class ProjectReview extends Component {
   };
 
   isValid() {
-   let descriptionError = false;
-   let projectTitleError = false;
-   let subjectProtectionError = false;
-   let editTypeError = false;
-   let editDescriptionError = false;
-   
-   if (this.state.projectType === "IRB Project" && this.isEmpty(this.state.formData.projectExtraProps.editDescription)) {
+    let descriptionError = false;
+    let projectTitleError = false;
+    let subjectProtectionError = false;
+    let editTypeError = false;
+    let editDescriptionError = false;
```

Comment:
There are a *lot* of white space changes. It would be simpler to review if we split those out into a single PR. I'm fine with this for this PR, but for the future, it would be easier to review :-)

Reply:
Sorry for that, Nadya did some code format in her commit. Will try not to do this again :)

Reply:
No worries, thank you!